client/components/Navigation.jsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
+import {Link, Switch}  from 'react-router-dom'
+import {BrowserRouter as Router, Route} from 'react-router-dom'
 import styled from 'styled-components';
 import Logo from 'Components/Logo';
 import { GREY, WHITE, YELLOW } from 'Styles/colors';
+import About from './About'
```

Comment:
please remove line 7 and 8. 
if you are not using about or download app components do not import. makes app do more work than it needs to.